lib/core/src/rcGlobal.cpp

```diff
@@ -134,33 +134,30 @@ int NumStructFileType = sizeof( StructFileTypeDef ) / sizeof( structFileTypeDef_
 
 /* valid keyWds for dataObjInp_t */
 validKeyWd_t DataObjInpKeyWd[] = {
```

Comment:
Where is this used?

Reply:
`msParam.cpp`: https://github.com/irods/irods/blob/d8117e2191186465198f67b116ac0d03045f5050/lib/core/src/msParam.cpp#L1440-L1460